internal/grpc/proto/kong/admin/model/v1/vault.proto

```diff
@@ -23,6 +23,9 @@ message Vault {
     string mount = 4;
     string kv = 5;
     string token = 6;
+    string auth_method = 7;
```

Comment:
Hmm, realized we're seemingly missing validation for many of these vault config fields... but not required to cover in this PR, just wanted to point it out.